WalletWasabi/WabiSabi/Backend/Rounds/RoundParameters.cs

```diff
@@ -113,4 +115,15 @@ public record RoundParameters
 
 	public Transaction CreateTransaction()
 		=> Transaction.Create(Network);
+
+	/// <returns>Min output amount that's economically reasonable to be registered with current network conditions.</returns>
+	/// <remarks>It won't be smaller than min allowed output amount.</remarks>
+	public Money CalculateMinReasonableOutputAmount()
+	{
+		var minEconomicalOutput = MiningFeeRate.GetFee(
+						Math.Max(
+							ScriptType.P2WPKH.EstimateInputVsize() + ScriptType.P2WPKH.EstimateOutputVsize(),
+							ScriptType.Taproot.EstimateInputVsize() + ScriptType.Taproot.EstimateOutputVsize()));
+		return Math.Max(minEconomicalOutput, AllowedOutputAmounts.Min);
+	}
```

Comment:
This is misplaced here, `RoundParameters` should remain a bag of parameters. I understand the idea but if we want to have this calculation about outputs then it would be better to move it to the OutputProvider where this is used and where makes much more sense. 

The same for `IsTaprootAllowed`.

Reply:
Thanks! The reason I did that is because they'll be accessed in 2 other places in the CoinJoinClient: https://github.com/zkSNACKs/WalletWasabi/pull/10830/files#diff-b4b0ed52d80da8067250690ccb013b4ba124c66ddbe1969236ac568759630031